mava/configs/logger/base_logger.yaml

```
@@ -4,7 +4,7 @@ base_exp_path: results # Base path for logging.
 use_console: True # Whether to log to stdout.
 use_tb: False # Whether to use tensorboard logging.
 use_json: False # Whether to log marl-eval style to json files.
-use_neptune: False  # Whether to log to neptune.ai.
+use_neptune: True  # Whether to log to neptune.ai.
```

Please reset to the default. 
```suggestion
use_neptune: False  # Whether to log to neptune.ai.
```